Doppler.AccountPlans/Controllers/AccountPlansController.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Doppler.AccountPlans.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Doppler.AccountPlans.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class HelloController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountName}/newplan/{planId}/calculate/{promocode?}")]
+        public string GetCalculateUpgradeCost([FromRoute] string accountName, [FromRoute] int planId, [FromRoute] string promocode = "doppler")
```

Comment:
why do you have a promocode by default?
```suggestion
        public string GetCalculateUpgradeCost([FromRoute] string accountName, [FromRoute] int planId, [FromQuery] string promocode = null)
```

Reply:
you're right, it should be null by default. As there is another task that is to create the endpoint itself, I did not think about it too much.
I change it.